index.bs

```diff
@@ -908,6 +908,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
             1. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
                 |authenticator| and [=set/remove=] it from |issuedRequests|.
 
+                Note: An [=authenticator=] may return some indication of "the user cancelled the entire operation".
+                Conveyance of such an indication is client platform-specific.
```

Comment:
I think you mean transport-specific - not platform-specific.

Reply:
Hrmm; I think Jeff means it's up to the user agent?

Reply:
whether such a "user cancelled entire operation" indication is conveyed successfully to the RP client-side JS, is specific, as I understand it, to the entire userAgent+OSplatform+transport+authenticator stack. That's what "client platform-specific" is attempting to convey.  

fyi/fwiw, "client platform" (and synonyms thereof) are among our currently undefined terms.  So making this stuff more clear may involved getting around to defining at least that term, seems to me.